testpackages.txt

```diff
@@ -22,6 +22,7 @@ github.com/getlantern/geolookup
 github.com/getlantern/golog
 github.com/getlantern/idletiming
 github.com/getlantern/keyman
+github.com/getlantern/lantern
```

Comment:
Adding this @oxtoacart to make sure our tests catch issues with breaking liblantern